core/src/test/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainerTest.java

```diff
@@ -75,4 +95,4 @@ public String getCode() {
             return code;
         }
     }
-}
+}
```

Comment:
```suggestion
}

```

EOF blank line is standard